themes/default/layouts/page/pulumi-up-2022.html

```diff
@@ -0,0 +1,62 @@
+{{ define "main" }}
+    <section id="overview" class="star-wars overflow-hidden">
+        <div class="h-full flex justify-center items-center">
+            <div class="title-animation">
+                <p class="title-part">Pulumi</p>
+                <div class="subtitle">The Cloud Awakens</div>
+                <p class="title-part">Up</p>
+            </div>
+        </div>
+
+        <div class="fade"></div>
+        <div class="crawl">
+             <div class="title">
+                <p>PulumiUP</p>
+                <h1>The Cloud Awakens</h1>
+            </div>
+
+            <p class="mb-24">
+                Turmoil has engulfed the cloud community. The techniques used to solve the
```

Comment:
I don't know when this is going live, but do we want to be talking about turmoil and factions right now?

Reply:
Updated the text to remove any references to turmoil or factions.